src/app/templates/CoordinateSectorCreator.html

```diff
@@ -0,0 +1,34 @@
+<div>
+  <form data-dojo-attach-point="formNode">
+    <div class="show" data-dojo-attach-point="ddNode">
+       <div class="form-group">
+         <label>Longitude (dd)</label>
+         <div class="input-group">
+          <span class="input-group-addon">-</span>
+          <input type="text" name="x" data-required="true" class="form-control" value=>
+        </div>
+      </div>
+      <div class="form-group">
+       <label>Latitude (dd)</label>
+       <input type="text" name="y" data-required="true" class="form-control" value=>
+     </div>
+     <div class="form-group">
+       <label>Beam Width (deg)</label>
+       <div class="input-group">
```

Comment:
I would remove this wrapper around the `input` that it contains since it seems to cause a strange styling:
![image](https://cloud.githubusercontent.com/assets/1326248/15324360/5d4afefc-1c03-11e6-8646-baa873657487.png)